karma.conf.js

```diff
@@ -14,7 +14,10 @@ module.exports = function(karma) {
 
 		browserify: {
 			debug: true
-		}
+		},
+
+		browserNoActivityTimeout: 500000,
```

Comment:
500000 (8.20 minutes) is quite a lot, we could start with 1 or 2 minutes. Can you also add a comment above these values to explain why and a link to the ticket/article/docs where you found that solution?

Reply:
Oops. I meant to set to 50000 not 500000. I've updated it to be 1 min and added a comment as suggested. Thanks for catching that